Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.swift

```diff
@@ -1,5 +1,45 @@
 import UIKit
 
+// MARK: View Model
+
+extension SavedRecipesTableViewCell {
+    struct ViewModel: Hashable, Equatable {
+
+        // MARK: - Properties
+
+        let id = UUID()
+
+        /// The title of the recipe
+        let title: String?
+
+        /// The  image of the recipe
+        let recipeImage: UIImage?
+
+        /// The rating of the recipe
+        let rating: Double?
+
+        /// The chef  who posted the recipe
+        let chefName: String?
+
+        /// The cooking Time of The recipe
+        let cookingTime: Int?
+
+        var toggleBookmark: () -> Void
+
+        func hash(into hasher: inout Hasher) {
+            hasher.combine(id)
+        }
+
+        static func == (lhs: SavedRecipesTableViewCell.ViewModel, rhs: SavedRecipesTableViewCell.ViewModel) -> Bool {
+            return  lhs.title == rhs.title &&
+            lhs.rating == rhs.rating &&
+            lhs.chefName == rhs.chefName &&
+            lhs.cookingTime == rhs.cookingTime &&
+            lhs.recipeImage == rhs.recipeImage
```

Comment:
```suggestion
            lhs.id == rhs.id
```

Reply:
<img width="1440" alt="Screenshot 2023-07-08 at 6 46 16 PM" src="https://github.com/motoon-eg/healthy/assets/45472327/17cb12c9-e6fa-4a82-bba9-03606d8cae56">

test failed when using lys.id == rhs.id only I searched for it the solution is ensure every property is equal

Reply:
@ahmdmhasn